jest.config.ts

```diff
@@ -7,6 +7,10 @@ const config: JestConfigWithTsJest = {
     transform: {
         "^.+\\.jsx?$": "babel-jest"
     },
+    transformIgnorePatterns: [
+        "/node_modules/(?!(date-fns/locale)/)",
+        "\\.pnp\\.[^\\\/]+$"
+    ],
```

Comment:
What is this for ?

Reply:
I forgot to delete it. It was to make jest work with dynamic imports of locale files (no more required thanks to use of built-in `Intl.DateTimeFormat` function)